pom.xml

```
@@ -17,7 +17,13 @@
             https://raw.githubusercontent.com/mate-academy/style-guides/master/java/checkstyle.xml
         </maven.checkstyle.plugin.configLocation>
     </properties>
```

```suggestion
    </properties>
    
```
Давай повернемо тут пустий рядок для покращення читабельності

---

Давай пофіксимо цей комент

---

Та наче запушив, в мене все ок показує..